hail_scripts/reference_data/combine.py

```diff
@@ -81,34 +89,31 @@ def get_ht(dataset: str, reference_genome: str):
         },
     )
     ht = ht.transmute(**get_enum_select_fields(config.get('enum_select'), ht))
-    return ht.select(**{field_name: ht.row.drop(*ht.key)}).distinct()
+    ht = ht.select_globals(
+        **{
+            f'{dataset}_globals': hl.struct(
+                path=config['path'],
+                version=ht.globals.get(
```

Comment:
idea here is to take the `version` off of the globals if it exists, then take it from the `config`.

Reply:
whats the situation where the existing global would be different from what is specified in the config?

Reply:
I think it'll only happen if 1) human doesn't put a version tag in the config or puts the incorrect one OR 2) we have a dataset with a static path but a changing version.  I think neither might come into play?

Reply:
I feel like since these are not expected cases its better to throw an error if we don't have the value we expect, but not have a graceful fallback mechanism for it

Reply:
👍 makes sense to me!